bgpd/bgp_attr.c

```diff
@@ -987,19 +987,21 @@ bgp_attr_malformed(struct bgp_attr_parser_args *args, uint8_t subcode,
 		return BGP_ATTR_PARSE_PROCEED;
 
 	/* Core attributes, particularly ones which may influence route
-	 * selection, should always cause session resets
+	 * selection, should be treat-as-withdraw.
```

Comment:
In function bgp_attr_check if mandatory well-known is missing as per RFC 7606 section 3 point d says MUST treat as "treat-as-withdraw""	but seems we are using BGP_ATTR_PARSE_ERROR. Any plan to change to retain as is?


       if (type) {
		flog_warn(EC_BGP_MISSING_ATTRIBUTE,
			  "%s Missing well-known attribute %s.", peer->host,
			  lookup_msg(attr_str, type, NULL));
		bgp_notify_send_with_data(peer, BGP_NOTIFY_UPDATE_ERR,
					  BGP_NOTIFY_UPDATE_MISS_ATTR, &type,
					  1);
		return BGP_ATTR_PARSE_ERROR;
	}

Reply:
Will replace it too.